### ISSUE TITLE
YOLOv3 Training but not Detecting

### ISSUE THREAD

**Issue by JasonACDsouza** [AlexeyAB/darknet#3761]
I am training on a dataset with ~90k images(640 x 640) with just a single class(person), running kmeans to find its own anchors, making required changes in filters in the .cfg file and changing class number/names in the .data/.classes files and run for an iteration of 18k with the following status:

Screenshot from 2019-08-08 11-49-53

So, the training starts at ~6000 and learns till I achieve a loss of ~0.7 and I stop training. But when I am trying to test it out on a sample video, it does not detect any person.
This is what my .cfg has,

batch=64
subdivisions=16
width=640
height=640
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.001
burn_in=500
max_batches=40000
policy=steps
steps=30000
scales=.1

What is exactly wrong?

**Comment by LukeAI** [comment]
can you share the chart.png ?

**Comment by JasonACDsouza** [comment]
I don't have the chart.png for it. I used the https://github.com/pjreddie/darknet repo

**Comment by JasonACDsouza** [comment]
> can you share the chart.png ?
I don't have the chart.png for it. I used the https://github.com/pjreddie/darknet repo

**Comment by varghesealex90** [comment]
Hello @JasonACDsouza ,

Not sure if this will help you. When you test can you edit the cfg 

From:
# Testing
#batch=1
#subdivisions=1
# Training
batch=64
subdivisions=16

**TO**

# Testing
batch=1
subdivisions=1
# Training
#batch=64
#subdivisions=16

**Comment by jfangah** [comment]
Did you train your model on coco dataset? You may need to check your category_id in your json file.
